File: 01 - Iniciante/1149.py
valores = input().split()
a = int(valores[0])
n = int(valores[1])

while n <= 0:
    valores = input().split()
    n = int(valores[1]) 

soma = 0
for i in range(n):
    soma += a
    a += 1

print(soma)

File: 01 - Iniciante/1149.cpp
#include <iostream>

int main() {
    int a, n;
    std::cin >> a >> n;

    while (n <= 0) {
        std::cin >> n; 
    }

    int soma = 0;
    for (int i = 0; i < n; ++i) {
        soma += a;
        ++a;
    }

    std::cout << soma << std::endl;
    return 0;
}